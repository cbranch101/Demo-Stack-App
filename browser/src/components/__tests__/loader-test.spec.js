import createTestContext from "react-cosmos-test/enzyme"

import fixture from "../__fixtures__/basic"

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

test("renders basic loader data", async () => {
    expect(getWrapper().html()).toMatchSnapshot()
})
