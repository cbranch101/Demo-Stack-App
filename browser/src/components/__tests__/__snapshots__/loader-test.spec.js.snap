// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders basic loader data 1`] = `"<div><div><!-- react-text: 3 -->Posts for : <!-- /react-text --><!-- react-text: 4 -->Clay<!-- /react-text --></div><ul><li>I'm a post</li></ul></div>"`;
